x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/tabular_page.tsx

```diff
@@ -57,38 +49,32 @@ export const TabularPage: React.FC<TabularPageProps> = ({ inferenceEndpoints })
   );
 
   const tableColumns: Array<EuiBasicTableColumn<InferenceEndpointUI>> = [
-    {
-      name: '',
-      render: ({ endpoint, deployment }: InferenceEndpointUI) =>
-        isLocalModel(endpoint) ? <DeploymentStatus status={deployment} /> : null,
-      align: 'center' as HorizontalAlignment,
-      width: '64px',
-    },
     {
       field: 'endpoint',
       name: i18n.ENDPOINT,
-      render: (endpoint: InferenceAPIConfigResponse) => {
+      render: (endpoint: string) => {
         if (endpoint) {
-          return <EndpointInfo endpoint={endpoint} />;
+          return <EndpointInfo inferenceId={endpoint} />;
         }
 
         return null;
       },
       sortable: true,
       truncateText: true,
+      width: '400px',
```

Comment:
nitpick(non-blocking): if possible can we calculate them as euiTheme base size * quotient.